ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/NtiUnderConsideration/Close.cshtml.cs

```diff
@@ -21,12 +21,16 @@ public class ClosePageModel : AbstractPageModel
 		private readonly INtiUnderConsiderationStatusesCachedService _ntiStatusesCachedService;
 		private readonly ILogger<ClosePageModel> _logger;
 
-		public int NotesMaxLength => 2000;
+		public  int NotesMaxLength => 2000;
 		public IEnumerable<RadioItem> NTIStatuses;
```

Comment:
Convert this to use the component then you get all the error formatting for free

Reply:
Changed